src/config.rs

```diff
@@ -0,0 +1,17 @@
+/// Default NATS server host
```

Comment:
Is a separate module really needed for this?

Especially, since you're importing all of these

Reply:
I would rather not try to prematurely optimize this and keep it simple for now